### ISSUE TITLE
timeline_widget breaks VFX Platform 2015 compatibility 

### ISSUE THREAD

**Issue by dramyankee** [PixarAnimationStudios/OpenTimelineIO#289]
With the update from PySide to PySide2 after beta version 0.7.1 the timeline widget is no longer compatible with VFX Platform 2015. Unfortunately we have not yet moved past 2015.

**Comment by jminor** [comment]
That's unfortunate. Can you check to see if switching to use Qt.py works for you? We're not allowed to include Qt.py in OTIO, but it should be pretty easy to try it out. https://github.com/mottosso/Qt.py

**Comment by jminor** [comment]
@dramyankee were you able to try Qt.py as a workaround for this?

**Comment by dramyankee** [comment]
Hi @jminor, yes that worked for me. Perhaps adding a try/except around the PySide2 import which falls through to attempting to import from Qt may be added?

**Comment by jminor** [comment]
Oh good. Did you have to update all of the `from PySide2 ...` statements to `from Qt ...`?

**Comment by dramyankee** [comment]
That's correct.

**Comment by melMass** [comment]
@jminor may I ask the reason not to go with `Qt.py`, it seems really suited to the project.

I'm planing on making a **fork with Qt.py**
if that's not planned by you guys, just waiting for a proper `1.0`

**Comment by ssteinbach** [comment]
@melMass this was the thread: https://github.com/PixarAnimationStudios/OpenTimelineIO/issues/73

**Comment by melMass** [comment]
@ssteinbach Thanks Stephan, what are the legal concerns ? [@motosso](https://github.com/mottosso) might help

**Comment by alatdneg** [comment]
I've got the same problem no access to PySide2, I tried using Qt.py but I get PySide.QtGui.QFontDatabase' has no attribute 'systemFont. I guess it's using something only suported in Qt5.

**Comment by mottosso** [comment]
> We're not allowed to include Qt.py in OTIO

Anything that could be changed in Qt.py to enable this? Could issue the source under public domain, which is pretty much what MIT is already.

**Comment by jminor** [comment]
Let me ask our legal folks for clarification on why they don’t want us using Qt.py.

**Comment by tokejepsen** [comment]
@jminor any update from legal folks about using Qt.py?

**Comment by mottosso** [comment]
My understanding is that legal confused Qt.py for PyQt (quite similar-sounding names, especially to the layman), which is under the GPL, and that they preferred to stick with PySide, which is LGPL, even though Qt.py is under MIT (which is more permissive than both GPL and LGPL).

**Comment by jminor** [comment]
No, nothing on this subject has changed. Sorry. We can ask again after OTIO transitions to ASWF - that might change the equation, but I don't know.
